### DISCUSSION TITLE
Put node back into maintenance mode?

### DISCUSSION THREAD

**Discussion by v1nsai** [siderolabs/talos#9193] [Q&A] [answered]
Is there a way to put a node back into maintenance mode without having to boot and wipe from USB or re-create a VM?

**Comment by smira** [comment, marked as answer]
Yes, you can do `talosctl reset --system-labels-to-wipe EPHEMERAL,STATE --reboot`

**Reply by kristvanbesien** [reply]
I have a cluster with 3 sbc, and when I do this the command hangs indefinetly. 

```
◲ ◱ watching nodes: [tl-ctrl1 tl-ctrl2 tl-ctrl3]
    * tl-ctrl1: events check condition met
    * tl-ctrl2: events check condition met
    * tl-ctrl3: unavailable, retrying...
```

What is going on?

**Reply by rothgar** [reply]
You'd need to provide logs from the node for us to understand what's happening. There could be a lot of things.

**Reply by kristvanbesien** [reply]
Eventually I ended up just flashing the nodes again...

**Comment by v1nsai** [comment]
Thanks!  I needed to add `--graceful=false` to get the last node in the cluster to reset but otherwise worked perfectly

**Comment by jfmatth** [comment]
I would add that the maintenance mode uses the version you initially booted with, which can sometimes throw you off if you're looking for the features from the boot device.

For example, I use Proxmox (i.e. QEMU) and if I boot with an ISO that supports that but not specify the right image, it can cause issues.

Just FYI
